.github/workflows/lint.yml

```diff
@@ -27,6 +30,9 @@ jobs:
       - name: Install project deps with pnpm
         run: |
           pnpm i
+      - name: Install Ruff
+        run: |
+          pip install ruff
```

Comment:
Can we use ruff-action? https://github.com/ChartBoost/ruff-action

Reply:
Well with the current approach it works locally for people to do `python ./tools/cross/format.py` if they have ruff installed, whereas if we used the action there wouldn't be a good local approach. I think @danlapid said he'd prefer to modify `format.py` over using ci-specific automation.

Reply:
`ruff-action` is used to install ruff. it is not used to execute ruff. right now, we will always use the latest ruff version.